tasks/fine_tuning_sbert/src/loops.py

```diff
@@ -212,13 +315,15 @@ def evaluate_using_sklearn(clf, model, train_sents, train_labels, test_sents, te
     print(classification_report(test_labels, clf_preds))
     numeric_preds = labels2numeric(clf_preds, label_names)
     numeric_test_labels = labels2numeric(test_labels, label_names)
-    evaluator = ModelEvaluator(label_names, y_true=numeric_test_labels, y_pred=numeric_preds)
-
-    print("Visualizing...")
-    out_path = f"{output_path}/{model_deets}" if output_path and model_deets else None
-    visualize_embeddings_2D(np.vstack(test_embs), test_labels, tsne_perplexity=50,
-                            output_path=out_path)
-    evaluator.plot_confusion_matrix(color_map='Blues', output_path=out_path)
+    evaluator = ModelEvaluator(
+        label_names, y_true=numeric_test_labels, y_pred=numeric_preds)
+
+    if not testing:
```

Comment:
Should we also store the CM, embedding visualization and classification report in wandb? or is that unnecessary? your call!

Reply:
I think that the CM and embedding visualization for the test set (inside the ``evaluate_using_sklearn`` function) can be stored in wandb. I made the adjustment for that and will test it. For the classification report (F1 accuracies), I think it is better to see and keep it in the notebook cell run itself.